blackbox/docs/sql/statements/create_table.txt

```diff
@@ -31,7 +31,7 @@ Synopsis
 
 where ``base_column_definition``::
 
-    column_name data_type [ column_constraint [ ... ] ]
+    column_name data_type [ column_constraint [ ... ] ]  [ storage_option ]
```

Comment:
`storage_option` is used here as placeholder for the whole expression and also below for the specific storage param.

I would rename this to something like storage_configuration or storage_option**s** , or the other one (https://github.com/crate/crate/pull/6583/commits/112c07cb9c73f01af9894761d49ac0da3754b900#diff-49b1eb6da27aeff61046466a2ac4697aR52) to just `option` or `param`